### PR TITLE
JWWEB-556: Migrate all texts to TTAG library

### PR DESCRIPTION
**Pull request by flexxnn**

### CODE REVIEW

**Comment by stcherenkov** [comment]
Please, don't forget that errors require string representation too, not only codes.

File: src/components/SingularTabBlockScreen/SingularTabBlockScreen.js [hunk 1]
```diff
@@ -14,18 +15,18 @@ const SingularTabBlockScreen = () => (
             color='white'
             iconSize='xlarge'
             image='bad-browser'
-            title='Jwallet supports only single tab'
+            title={t`Jwallet supports only single tab`}
             description={[
-              'Jwallet can be open only in one tab simultaneously.',
-              'Please reload this tab to continue using it.',
+              t`Jwallet can be open only in one tab simultaneously.`,
```
Comment (stcherenkov) [inline src/components/SingularTabBlockScreen/SingularTabBlockScreen.js]: FYI: double line

File: src/components/TransactionsList/Empty/Empty.js [hunk 2]
```diff
@@ -14,10 +15,10 @@ function TransactionsListEmpty({ isFiltered }: Props) {
       color='gray'
       image='screen-reload'
       description={isFiltered ? [
-        'There are no items to display',
+        t`There are no items to display`,
       ] : [
-        'Looks like you haven\'t made',
-        'any transactions yet.',
+        t`Looks like you haven't made`,
```
Comment (stcherenkov) [inline src/components/TransactionsList/Empty/Empty.js]: Double line

File: src/routes/DigitalAssets/routes/Grid/DigitalAssetsGridViewContainer.js [hunk 3]
```diff
@@ -160,7 +161,7 @@ function mapStateToProps(state: AppState) {
   )
 
   const assetsWithBalance: DigitalAssetWithBalance[] = getDigitalAssetsWithBalance(
-    flattenDigitalAssets(assets),
+    flatten(assets),
```
Comment (stcherenkov) [inline src/routes/DigitalAssets/routes/Grid/DigitalAssetsGridViewContainer.js]: That's not about ttag. Why did you change it?

File: src/routes/Wallets/routes/Addresses/WalletsAddressesView.js [hunk 4]
```diff
@@ -53,13 +54,13 @@ class WalletsAddressesView extends Component<Props> {
...
           color='white'
-          title='Mnemonic addresses'
+          title={t`Mnemonic addresses`}
         />
         <div className='content'>
           <WalletViewTitle
             data={[
-              'To enhance your privacy, the wallet can contain as many addresses',
-              'as you need. Please, choose one to continue',
+              t`To enhance your privacy, the wallet can contain as many addresses`,
```
Comment (flexxnn) [inline src/routes/Wallets/routes/Addresses/WalletsAddressesView.js]: double line

Comment (stcherenkov) [inline src/routes/Wallets/routes/Addresses/WalletsAddressesView.js]: Double lines

File: src/routes/Wallets/routes/Create/WalletsCreateView.js [hunk 5]
```diff
@@ -84,14 +85,14 @@ class WalletsCreateView extends Component<Props> {
               onChangePasswordHint={changePasswordHintInput}
               onChangePasswordConfirm={changePasswordConfirmInput}
               title={[
-                'You will use this password to unlock and transfer your funds.',
-                'Keep it secure!',
+                t`You will use this password to unlock and transfer your funds.`,
```
Comment (stcherenkov) [inline src/routes/Wallets/routes/Create/WalletsCreateView.js]: Multiline

File: src/routes/Wallets/routes/Import/WalletsImportView.js [hunk 6]
```diff
@@ -94,13 +95,13 @@ class WalletsImportView extends Component<Props> {
               onChangeData={changeDataInput}
               onChangeDerivationPath={changeDerivationPathInput}
               title={[
-                'Insert the private key or backup phrase to import. Also, you can import',
-                'an address in the read-only mode.',
+                t`Insert the private key or backup phrase to import. Also, you can import`,
```
Comment (stcherenkov) [inline src/routes/Wallets/routes/Import/WalletsImportView.js]: Multiline

File: src/routes/Wallets/routes/Import/WalletsImportView.js [hunk 7]
```diff
@@ -112,14 +113,14 @@ class WalletsImportView extends Component<Props> {
               onChangePasswordHint={changePasswordHintInput}
               onChangePasswordConfirm={changePasswordConfirmInput}
               title={[
-                'You will use this password to unlock and transfer your funds.',
-                'Keep it secure!',
+                t`You will use this password to unlock and transfer your funds.`,
```
Comment (stcherenkov) [inline src/routes/Wallets/routes/Import/WalletsImportView.js]: Multiline

File: src/routes/DigitalAssets/routes/Send/components/Confirm/Card/Card.js [hunk 8]
```diff
@@ -47,27 +48,28 @@ function DigitalAssetsSendConfirmCard({
...
   return (
     <div className='digital-assets-send-confirm-card'>
       <div className='content'>
         {isPotentiallyFail &&
         <div className='willfail'>
-          Your transaction does not pass one or more check-ups,
-          so we assume <b>it will most likely fail</b>. Please make sure you
-          know what you are doing before proceeding
+          { // eslint-disable-next-line max-len
+            jt`Your transaction does not pass one or more check-ups, so we assume ${boldWillFail}. Please make sure you know what you are doing before proceeding`}
```
Comment (IViol) [inline src/routes/DigitalAssets/routes/Send/components/Confirm/Card/Card.js]: big line could be splitted

File: src/utils/analytics/ga.js [hunk 9]
```diff
@@ -37,6 +37,7 @@ export const ga = (...args) => {
     try {
       window.ga(...args)
     } catch (err) {
+      // #TODO: TTAG???
```
Comment (stcherenkov) [inline src/utils/analytics/ga.js]: Nope, technical error

File: src/utils/analytics/ga.js [hunk 10]
```diff
@@ -53,6 +54,7 @@ export const gaSendEvent = (...args) => {
 
 export const gaSetUserMetric = (metric, value) => {
   if (!metric) {
+    // #TODO: TTAG???
```
Comment (stcherenkov) [inline src/utils/analytics/ga.js]: Nope, technical error

File: src/utils/analytics/ga.js [hunk 11]
```diff
@@ -61,6 +63,7 @@ export const gaSetUserMetric = (metric, value) => {
 
 export const gaSetUserDimension = (dimension, text) => {
   if (!dimension) {
+    // #TODO: TTAG???
```
Comment (stcherenkov) [inline src/utils/analytics/ga.js]: Nope, technical error
